addons/skin.estuary/xml/Variables.xml

```diff
@@ -738,259 +738,124 @@
-		<value condition="String.IsEqual(ListItem.AudioCodec,wmapro)">WMA-PRO</value>
-		<value condition="String.IsEqual(ListItem.AudioCodec,wmav2)">WMA</value>
+	<variable name="AudioCodecVar">
+		<value condition="String.IsEqual(ListItem.AudioCodec,aac) | String.IsEqual(VideoPlayer.AudioCodec,aac) | String.IsEqual(MusicPlayer.AudioCodec,aac)">AAC</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,aac_latm) | String.IsEqual(VideoPlayer.AudioCodec,aac_latm) | String.IsEqual(MusicPlayer.AudioCodec,aac_latm)">AAC</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,ac3) | String.IsEqual(VideoPlayer.AudioCodec,ac3) | String.IsEqual(MusicPlayer.AudioCodec,ac3)">Dolby Digital</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,aif) | String.IsEqual(VideoPlayer.AudioCodec,aif) | String.IsEqual(MusicPlayer.AudioCodec,aif)">AIFF</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,aifc) | String.IsEqual(VideoPlayer.AudioCodec,aifc) | String.IsEqual(MusicPlayer.AudioCodec,aifc)">AIFF</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,aiff) | String.IsEqual(VideoPlayer.AudioCodec,aiff) | String.IsEqual(MusicPlayer.AudioCodec,aiff)">AIFF</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,alac) | String.IsEqual(VideoPlayer.AudioCodec,alac) | String.IsEqual(MusicPlayer.AudioCodec,alac)">ALAC</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,ape) | String.IsEqual(VideoPlayer.AudioCodec,ape) | String.IsEqual(MusicPlayer.AudioCodec,ape)">APE</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,avc) | String.IsEqual(VideoPlayer.AudioCodec,avc) | String.IsEqual(MusicPlayer.AudioCodec,avc)">AVC</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,cdda) | String.IsEqual(VideoPlayer.AudioCodec,cdda) | String.IsEqual(MusicPlayer.AudioCodec,cdda)">CDDA</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,dca) | String.IsEqual(VideoPlayer.AudioCodec,dca) | String.IsEqual(MusicPlayer.AudioCodec,dca)">DTS</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,dolbydigital) | String.IsEqual(VideoPlayer.AudioCodec,dolbydigital) | String.IsEqual(MusicPlayer.AudioCodec,dolbydigital)">Dolby Digital</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,dts) | String.IsEqual(VideoPlayer.AudioCodec,dts) | String.IsEqual(MusicPlayer.AudioCodec,dts)">DTS</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,dtshd_hra) | String.IsEqual(VideoPlayer.AudioCodec,dtshd_hra) | String.IsEqual(MusicPlayer.AudioCodec,dtshd_hra)">DTSHD-HRA</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,dtshd_ma) | String.IsEqual(VideoPlayer.AudioCodec,dtshd_ma) | String.IsEqual(MusicPlayer.AudioCodec,dtshd_ma)">DTSHD-MA</value>
+		<value condition="String.IsEqual(ListItem.AudioCodec,dtsma) | String.IsEqual(VideoPlayer.AudioCodec,dtsma) | String.IsEqual(MusicPlayer.AudioCodec,dtsma)">DTSHD-MA</value>
```

Comment:
Not directly related to new audio formats or this PR but maybe change DTSHD-MA style to DTS-HD MA:

```
DTSHD-MA  --> DTS-HD MA
DTSHD-HRA --> DTS-HD HRA
```

Reply:
Yeah that makes sense, done.